contracts/base/ERC1155/ERC1155.sol

```diff
@@ -0,0 +1,255 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+contract ERC1155 {
```

Comment:
I feel like we should put these raw EIP implementations in another folder, the reason is that we want toi guide people to import the right thing in their contracts. If i see both `ERC1155` and `ERC1155Base` available side by side i would get confused which one to use at first glance.

what about `@thirdweb-dev/contracts/base/baseName` as our path that people will import (drop the intermediate folder)